include/pros/misc.h

```diff
@@ -721,6 +721,26 @@ double battery_get_capacity(void);
  */
 int32_t usd_is_installed(void);
 
+/******************************************************************************/
+/**                              Date and Time                               **/
+/******************************************************************************/
+
+extern const char* baked_date;
+extern const char* baked_time;
+
+typedef struct {
+	uint16_t year; // Year - 1980
+	uint8_t day;
+	uint8_t month; // 1 = January
+} date_s_t;
+
+typedef struct {
+	uint8_t hour;
+	uint8_t min;
+	uint8_t sec;
+	uint8_t sec_hund; // hundredths of a second
+} time_s_t;
```

Comment:
may not be necessary for this PR but we gotta doxygen comment these before release.